src/commands/initProjectForVSCode/detectProjectLanguage.ts

```diff
@@ -24,7 +24,10 @@ export async function detectProjectLanguage(context: IActionContext, projectPath
             if (await isTypeScriptProject(projectPath)) {
                 detectedLangs.push(ProjectLanguage.TypeScript);
             } else {
-                detectedLangs.push(ProjectLanguage.JavaScript);
+                // don't add JavaScript if TypeScript is already detected
```

Comment:
> if TypeScript is detected (tsconfig found, etc.)

I don't think it's checking for tsconfig right now, but that's an interesting idea

Reply:
My bad, I thought we were doing that. Adding a check for a tsconfig would also satisfy these tests since a tsconfig is created upon initializing the test folder.